src/ts/browserutils.ts

```diff
@@ -39,6 +39,38 @@ export class BrowserUtils {
     return navigator && navigator.userAgent && navigator.platform === 'MacIntel';
   }
 
+  static get isHisense(): boolean {
+    if (!this.windowExists()) {
+      return false;
+    }
+    return navigator && navigator.userAgent && /Hisense/.test(navigator.userAgent);
+  }
+
+  static get isPlayStation(): boolean {
```

Comment:
nit: we're only testing for `Playstation 5` in the UA, so the function name should reflect that.

Reply:
Changed the check in d6111066 to check for PlayStation in general, not PlayStation 5 in particular.